packages/select/src/select.vue

```diff
@@ -64,7 +64,7 @@
         v-model="query"
         @input="debouncedQueryChange"
         v-if="filterable"
-        :style="{ width: inputLength + 'px', 'max-width': inputWidth - 42 + 'px' }"
```

Comment:
When `collapse-tags` is set to `true`, there are some problems.
![kapture 2018-08-16 at 11 25 36](https://user-images.githubusercontent.com/12947422/44185722-c1bdee00-a147-11e8-97ea-b849bf17e1ab.gif)

Maybe you should not modify this line.

Reply:
@ziyoung now check.

Reply:
@akki-jat have you fixed it?
![kapture 2018-10-09 at 15 36 08](https://user-images.githubusercontent.com/12947422/46653734-67865a00-cbd9-11e8-96c9-b01188b6861b.gif)

Reply:
@ziyoung I have fixed it. Why? you still getting this behaviour?

Reply:
@ziyoung No, this was just **Error Reproduction Link**

Reply:
@akki-jat I mean you can use this example to test in your development environment.

Reply:
@ziyoung sorry I misunderstood what you said and I also test using that example and its working fine. 
### And you saying put calculating height code in timer?
I don't think it is appropriate to call a code if its not needed.

Reply:
@akki-jat I tried it again and found that the problem still exists. 

OS/Browsers version: Mac OS X 10_13_6/Chrome/69.0.3497.100

Reply:
@ziyoung  I don't know what is happening here, is it specific to mac? Because I tested it multiple times on windows 10 / chrome & its working perfectly fine.

Reply:
@wacky6 / @jikkai can anyone test it? Because it is working fine on my PC. Please check it.

Reply:
Confirm it has issue when `multiple`, `collapse-tags` and `filterable` are set.

See: https://jsfiddle.net/0fj6tpvo/1/
Built against https://github.com/akki-jat/element/commit/b77e59367a8b870f6a6186ffe01a5d9ab802e2c4
Tested on macOS 10.13.6, Chrome 69

Reply:
@wacky6 / @ziyoung test it again after taking latest.

Reply:
@ziyoung this problem caused by **resetInputHeight()** function. When we remove a tag this function is call to determine the input height but when this method calculates the height the **( +1 )** tag is not removed ( even if there are no extra selection ). So, when it calculates height it also takes +1 tag into consideration.

Note: UI updates after **resetInputHeight()** function

Reply:
@akki-jat @ziyoung I optimize `ElTag` component https://github.com/ziyoung/element/blob/dev/packages/tag/src/tag.vue It works fine on my computer.